modules/utxo-lib/test/bitgo/wallet/util.ts

```diff
@@ -14,17 +14,17 @@ export function mockOutputId(vout: number): string {
   });
 }
 
-export function mockWalletUnspent(
+export function mockWalletUnspent<TNumber extends number | bigint = number>(
   network: Network,
   {
     chain = 0 as ChainCode,
     index = 0,
-    value = defaultTestOutputAmount,
+    value = defaultTestOutputAmount as TNumber,
```

Comment:
instead of casting, it is probably better to not provide a default argument here at all

Reply:
moved value out of curly braces, turned into a regular arg